jekyll/_cci2/language-python.md

```diff
@@ -20,78 +20,74 @@ We maintain a reference Python Django project to show how to build Django on Cir
 
-## Alternative: Python Flask Demo Reference Project
+Refer to the [Project Walkthrough]({{ site.baseurl }}/2.0/project-walkthrough/) for a sample Flask application: <https://github.com/CircleCI-Public/circleci-demo-python-flask>.
 
-The [Project Walkthrough]({{ site.baseurl }}/2.0/project-walkthrough/) uses a Flask application: <https://github.com/CircleCI-Public/circleci-demo-python-flask>
-
-## Pre-built CircleCI Docker images
+## Pre-built CircleCI Docker Images
 
 We recommend using a CircleCI pre-built image that comes pre-installed with tools that are useful in a CI environment. You can select the Python version you need from Docker Hub: <https://hub.docker.com/r/circleci/python/>. The demo project uses an official CircleCI image.
 
-Database images for use as a secondary 'service' container are also available.
+Database images for use as a secondary 'service' container are also available on Docker Hub in the `circleci` directory.
 
-## Build the Demo Django Project Yourself
+## Build the Demo Django Project 
 
 A good way to start using CircleCI is to build a project yourself. Here's how to build the demo project with your own account:
 
-1. Fork the project on GitHub to your own account
-2. Go to the [Add Projects](https://circleci.com/add-projects) page in CircleCI and click the Build Project button next to the project you just forked
+1. Fork the project on GitHub to your own account.
+2. Go to the [Add Projects](https://circleci.com/add-projects) page in CircleCI and click the Build Project button next to the project you just forked.
 3. To make changes you can edit the `.circleci/config.yml` file and make a commit. When you push a commit to GitHub, CircleCI will build and test the project.
 
 ## Config Walkthrough
 
-We always start with the version:
+Start with the version:
 
 ```YAML
-version: 2
+version: 2 # use CircleCI 2.0
 ```
 
-Next, we have a `jobs` key. Each job represents a phase in your Build-Test-Deploy (BTD) process. Our sample app only needs a `build` job, so everything else is going to live under that key.
+First, specify a `jobs` key. Each job represents a phase in your Build-Test-Deploy (BTD) process. The sample app only needs a `build` job, so all other configuration will be nested under that key.
 
-We need to specify a working directory container images for this build in `docker` section:
+Specify a working directory and container images for this build in `docker` section:
 
 ```YAML
 ...
-jobs:
-  build:
-    working_directory: ~/circleci-demo-python-django
-    docker:
-      - image: circleci/python:3.6.1
-      - image: circleci/postgres:9.6.2
-        environment:
-          POSTGRES_USER: root
+jobs: # a collection of steps
+  build: # runs not using Workflows must have a `build` job as entry point
```

Comment:
The previous comment defined **what** the key was. This comment is saying that the key is required.

If we're going to add comments to every line of configuration, we should use them consistently.

Reply:
I think we just want explanatory text where it is helpful (every config will have a slightly different set of comments, they are free form).